### ISSUE TITLE
Missed square brackets in JS (JS syntax error) (v15.0)

### ISSUE THREAD

**Issue by danieldegtyarev** [bridgedotnet/Bridge#1821]
http://deck.net/4b3f1386c75fa311b4c88a07a2dfc7e5

The sample above generates 

```js
x."Demo$IInterface$1$Demo$TT$1$" + Bridge.getTypeAlias(T) + "$act"(new (Demo.TT$1(T))()); 
```

**Comment by geoffreymcgill** [comment]
Hi @danieldegtyarev Thanks for the report and excellent Deck demonstrating how to reproduce the issue. We are investigating and will try to fix this right away.

**Comment by danieldegtyarev** [comment]
Thank you.
Similar syntax error: http://deck.net/77cd1ad8802e89bc3b5039a61127c50f
(var x = foo.getA().Demo$IBar$1$Demo.Xxx$getValue(); at line 17)
I dont sure if it is the same

**Comment by geoffreymcgill** [comment]
This defect has been fixed and will be released as NuGet package **Bridge 15.0.1**. 

Both your Decks above now compile properly. 

Thanks again for reporting the issue.
